### DISCUSSION TITLE
기술블로그 반영에 기준이 있나요?

### DISCUSSION THREAD

**Discussion by Hoonja** [techcrontab/techcrontab#3] [Q&A]
1. 특정 기준을 만족해야 포함되는 것인지
2. 제보도 받는지
3. 국내 한정인지
... 궁금합니다.

**Comment by leechungkyu** [comment]
@Hoonja 님 안녕하세요 :-)

1. 특별히 기준은 없습니다. 기준을 생각한 적도 없는 것 같네요. 쓱 들어가봐서 1년 내외로 신규글이 없으면 추가하지 않았던거 같아요.
2. 예 제보해주세요.
3. 국내기업 한정이요. 한글컨텐츠 기업이라고 해도 맞겠네요.
해외기업은 별도 페이지로 만들까합니다.
p.s 자본은 빼고요 ㅋㅋ
